recipes/pyserini/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.11.0.0" %}
+
+package:
+  name: pyserini
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyserini/pyserini-{{ version }}.tar.gz
+  sha256: 4ff93e607f570f0643321cb073aa571ee298a7eec20b21c4e13812e16a75a65b
+
+build:
+  # noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - openjdk >=11
+    - pip
+  run:
+    - python
+    - cython >=0.29.21
+    - faiss >=1.6.5
```

Comment:
@MXueguang
Note, this is intentionally not `faiss-cpu`; conda-forge has a different naming scheme than the pytorch channel, which makes it possible for this recipe to depend on faiss independently of the CPU or GPU version (`faiss-cpu` & `faiss-gpu` are still provided for compatibility...).